Makefile

```diff
@@ -18,7 +18,7 @@
 # IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-MOCKGENERATE := go run github.com/golang/mock/mockgen@v1.7.0-rc.1
+MOCKGENERATE := go run go.uber.org/mock/mockgen@v0.4.0
 GINKGO := go run github.com/onsi/ginkgo/ginkgo@v1.16.5
```

Comment:
Please verify if the `-race` flag is being used on tests. Apparently it is, but I could not easily find what the `-r` flag stands for.